docs/authoring-components.md

```diff
@@ -738,7 +733,7 @@ class MDCNewComponentFoundation extends MDCFoundation {
 ```
 
 The primary purpose of this is so that our components can interoperate with aspects of Google's
-front-end infrastructure, such as Closure Stylesheets' [CSS Class Renaming](https://github.com/google/closure-stylesheets#renaming) mechanism. In addition, it
```

Comment:
This comment is not made irrelevant by the TypeScript conversion.

Reply:
ok I'll add this back